timberware/cinesync
# Closes #70: improve profile page

## Issue

Closes #70 

## Type of Change



## Description
This PR adds a few changes to the profile page: 
- responsiveness
- user stats
- uploading profile image.

**Note**
There's still a small issue with uploading a profile image. Images are usually successfully uploaded, but the profile page doesn't automagically refresh to display the uploaded image, so manual refresh is needed.
This issue will be addressed in a follow-up PR.

## Testing the PR
- Run migration on local BE
- Run BE and FE
- Assert that responsiveness, stats and uploading a profile image work as expected

## Checklist
